plugins/inputs/fireboard/README.md

```diff
@@ -0,0 +1,56 @@
+# Fireboard Input Plugin
+
+The fireboard plugin gathers the real time temperature data from fireboard
+thermometers.  In order to use this input plugin, you'll need to sign up
+to use their REST API, you can find more information on their website
+here [https://docs.fireboard.io/reference/restapi.html]
+
+### Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage <plugin-name>`.
+
+```toml
+[[inputs.fireboard]]
+  ## Specify auth token for your account
+  ## https://docs.fireboard.io/reference/restapi.html#Authentication
+  # authToken = "b4bb6e6a7b6231acb9f71b304edb2274693d8849"
+  ## You can override the fireboard server URL if necessary
+  # URL = https://fireboard.io/api/v1/devices.json
+```
+
+#### authToken
+
+In lieu of requiring a username and password, this plugin requires the
+authToken that you can generate using the Fireboard REST API as described
+in their docs [https://docs.fireboard.io/reference/restapi.html#Authentication]
+
+#### URL
+
+While there should be no reason to override the URL, the option is available
+in case Fireboard changes their site, etc.
+
+### Metrics
+
+The Fireboard REST API docs have good examples of the data that is available,
+currently this input only returns the real time temperatures. Temperature 
+values are included if they are less than a minute old.
+
+- fireboard
+  - tags:
+    - channel
+    - scale (1=celcius; 2=farenheit)
```

Comment:
Let's use the scale name instead of the integer: `scale=celcius`

Reply:
I do this now in fireboard.go GatherTemps() line 123/130. Would appreciate feedback on if this is the best way to do it.

Reply:
The worry with this method is that if somehow the data is not 0-2, then it will cause a panic.  I'd probably make a new function and use switch:
```go
func scale(n int) string {
	switch n {
	case 1:
		return "celcius"
	case 2:
		return "farenheit"
	default:
		return ""
	}
}
```

Reply:
Thanks. Implemented as a function.